tooling/build/scripts/seed-from-repo/index.ts

```diff
@@ -0,0 +1,153 @@
+import * as fs from "fs"
+import * as path from "path"
+import { performance } from "perf_hooks"
+import * as dotenv from "dotenv"
+import { Client } from "pg"
+
+dotenv.config()
+
+const DATABASE_URL = process.env.DATABASE_URL
+const SCHEMA_DIR = process.env.SCHEMA_DIR // Absolute path to the "schema" directory
+const SITE_ID = Number(process.env.SITE_ID) // Ensure SITE_ID is set in the environment
+const PUBLISHED_BY = "x" // Set the user ID for the publishedBy field
+
+async function main() {
+  if (!SCHEMA_DIR) {
+    console.error("Error: SCHEMA_DIR environment variable is not set.")
+    return
+  }
+
+  if (!SITE_ID) {
+    console.error("Error: SITE_ID environment variable is not set.")
+    return
+  }
+
+  const client = new Client({
+    connectionString: DATABASE_URL,
+  })
+
+  const start = performance.now() // Start profiling
+
+  try {
+    await client.connect()
+
+    const siteExists = await ensureSiteExists(client, SITE_ID)
+    if (!siteExists) {
+      console.error(`Error: Site with ID ${SITE_ID} does not exist.`)
+      return
+    }
+
+    await seedDatabase(client)
+  } finally {
+    await client.end()
+    const end = performance.now() // End profiling
+    console.log(`Database seeding completed in ${(end - start) / 1000} seconds`)
```

Comment:
nit - wondering if `DATABASE_URL` is invalid and `client.connect()` fails, will this console.log statement be a false positive?